source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -54,7 +54,7 @@ struct TextConfig
     ForegroundColor color = ForegroundColor::Default;
     bool isSubtle = false;
     bool wrap = true;
-    unsigned int maxWidth = 150;
+    unsigned int maxWidth = ((unsigned int)((unsigned int)~0));
```

Comment:
>((unsigned int)((unsigned int) [](start = 28, length = 30)

What does the double casting do? 

Would std::numeric_limits<unsigned int>::max() work here?

Reply:
Left over from development. Removed the extra cast. Tried std:numeric_limits<unsigned int>::max() but it seems that a function call in a struct definition doesn't work.